express/parsers/apps/espresso/settings.py

```diff
@@ -180,3 +181,8 @@
         "output_type": "float",
     },
 }
+
+HUBBARD_REGEX = {
```

Comment:
These seem much more generic (can be reused for many other engines) than what would be appropriate to use in this file, let's put to top-level settings?